bp_fe/src/include/bp_fe_decompress.vh

```diff
@@ -0,0 +1,247 @@
+`ifndef BP_FE_DECOMPRESS_VH
+`define BP_FE_DECOMPRESS_VH
+
+`ifndef BSG_DEFINES_V
+`define BSG_DEFINES_V
+`include "bsg_defines.v"
+`endif
+
+`ifndef BP_COMMON_FE_BE_IF_VH
+`define BP_COMMON_FE_BE_IF_VH
+`include "bp_common_fe_be_if.vh"
+`endif
+
+`define fe_decompress_s bp_fe_queue_s
+`define decompress_fe_s bp_fe_queue_s
+
+`define fe_decompress_width(vaddr_width_p,branch_metadata_fwd_width_p) \
```

Comment:
use "_mp" for macro parameter suffix (same applies for all files in pull request)